charts/pulsar-admin-console/templates/tests/run-pulsar.yaml

```diff
@@ -0,0 +1,46 @@
+#
+#  Copyright 2022 DataStax, Inc.
```

Comment:
Nit: I think we want 2023 here or maybe we could use the other license headers that don't have the date. I am not sure how much it matters.

Reply:
changed to non year-dependant license. Anyway the cass-operator doesn't have the license now so it's not a big deal